index.bs

```diff
@@ -190,6 +190,8 @@ There may be multiple [=WebTransport sessions=] on one [=connection=], when pool
 
 To <dfn for=session>establish</dfn> a [=WebTransport session=], follow [[!WEB-TRANSPORT-HTTP3]]
 [section 3.3](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-3.3).
+When establishing a session, the client MUST NOT provide any [=credentials=]
+via HTTP headers.
```

Comment:
Adam mentioned client certificates.  I would remove "via HTTP headers" and then be more explicit and explain that this means "no cookies", "no authorization", and "no client auth".

Reply:
If this ends up referencing https://fetch.spec.whatwg.org/#credentials removing "via HTTP headers" is probably sufficient.

Reply:
+1